s2core/src/main/scala/com/kakao/s2graph/core/Graph.scala

```diff
@@ -309,13 +312,14 @@ object Graph {
   } get
 }
 
-class Graph(_config: Config)(implicit ec: ExecutionContext) {
+class Graph(_config: Config)(implicit val ec: ExecutionContext) {
```

Comment:
@SteamShon I thought it necessary for test code. but now it's not necessary, i'll remove